ministryofjustice/correspondence_tool_staff
# CT-2761 circle CI

## Description


## Self-review checklist


### Screenshots

![image](https://user-images.githubusercontent.com/22935203/86119800-0c8e4800-bacb-11ea-9cc5-9e3306eb6333.png)


### Related JIRA tickets

https://dsdmoj.atlassian.net/browse/CT-2761

### Deployment

Deploy using Circle CI! (If it fails then switch to manual) - Note no DB stuff done, no migrations

### Manual testing instructions
Try deploying your builds using Circle CI once merged. But before that, you can rebase of this branch if you want to try to deploy. Also check workflow suits you and check steps on Circle CI.
